model/model.wl

```diff
@@ -389,17 +382,104 @@ evaluateScenario[state_, fitParams_, sims_, stateParams_, scenario_]:=Module[{
     deciles,
     containmentTime,
 	hospitalOverloadTime,
-	icuOverloadTime
+	icuOverloadTime,
+	sims,
+	testSim,
+	sim,
+	setDistancing,
+	rawSimResults,
+	paramExpected
     },
-    
-	distance[t_] := stateDistancing[state, scenario, t];
-	
-	(* do one solution with the mean param values for the estimate *)
-	{sol,evts}=CovidModel[
-	fitParams["r0natural"],
+  
+    Clear[distancing, pfunODE2];
+    distancing[t_] := stateDistancing[state, scenario["id"], t];
+    Clear[Sq,Eq,ISq,RSq,IHq,HHq,RHq,RepHq,Iq,ICq,EHq,HCq,CCq,RCq,Deaq,PCR,est];
+Clear[r0natural,daysUntilNotInfectiousOrHospitalized,daysFromInfectedToInfectious,daysToLeaveHosptialNonCritical,pPCRNH,pPCRH,daysTogoToCriticalCare,daysFromCriticalToRecoveredOrDeceased,fractionOfCriticalDeceased,importtime,importlength,initialInfectionImpulse,tmax,pS,pH,pC,containmentThresholdCases,icuCapacity,hospitalCapacity];
```

Comment:
Why clearing all these variables? Could we instead defining them as module scope in the module definition?

Reply:
I was getting weird issues where the definition of the parametric ode wouldn't respect things like the distancing function changing without defining these in the module and clearing. Maybe they could just be module scope though